### PR TITLE
fix(xyflow) fitViewOptions

### PR DESCRIPTION
**Pull request by peterkogo**

### CODE REVIEW

**Comment by moklick** [comment]
#3442 is also part of this PR

**Comment by peterkogo** [comment]
@moklick ready to merge

File: packages/system/src/utils/graph.ts [hunk 1]
```diff
@@ -171,7 +171,7 @@ export function fitView<Params extends FitViewParamsBase<NodeBase>, Options exte
   options?: Options
 ) {
   const filteredNodes = nodes.filter((n) => {
-    const isVisible = options?.includeHiddenNodes ? n.width && n.height : !n.hidden;
+    const isVisible = n.width && n.height && (options?.includeHiddenNodes || !n.hidden);
```
Comment (moklick) [inline packages/system/src/utils/graph.ts:174]: good catch :)

File: examples/svelte/src/routes/overview/+page.svelte [hunk 2]
```diff
@@ -131,8 +131,16 @@
...
+	fitViewOptions={{
+		padding: 0.1,
+		nodes: [
+			{
+				id: '1'
+			}
+		]
+	}}
+	minZoom={0}
+	maxZoom={Infinity}
```
Comment (moklick) [inline examples/svelte/src/routes/overview/+page.svelte:143]: could you use spaces here as well?

**Pull request merged**